packages/tailwind-composer/src/setupTests.ts

```
@@ -0,0 +1,40 @@
+        return this.utils.matcherHint(
+          `${this.isNot ? '.not' : ''}.toHaveClass`,
+          'element',
+          this.utils.printExpected(expected.join(' ')) +
+            '\n\n' +
+            this.utils.printDiffOrStringify(
+              expected,
+              received,
+              'Expected',
+              'Received',
+              this.expand !== false
+            )
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)